7_Search_and_filter.md

```diff
@@ -21,6 +22,26 @@ Add support for:
 * finding sales by product or category name;
 * finding purchases by product or category name.
 
+## Front-end testing
+
+Add some more DalekJS tests. You'll need some new methods like `type` and `submit`.
```

Comment:
@avermeulen Cool, thanks. I've updated it here: https://github.com/codex-academy/Nelisa/pull/16/commits/86bb6a35ee005a641c100314c43e3679e31c00f1